src/useIsStickyStuck/useIsStickyStuck.ts

```diff
@@ -0,0 +1,36 @@
+import { Ref, computed, onMounted, ref } from 'vue'
+import { MaybeRef } from '@/types/maybe'
+import { useIntersectionObserver } from '@/useIntersectionObserver'
+
+type UseIsStickyStuckOptions = {
+  rootMargin?: string,
+  boundingElement?: HTMLElement | Ref<HTMLElement | undefined>,
+}
+
+export function useIsStickyStuck(
```

Comment:
I don't love this name but I couldn't think of anything better. So I asked chat-GPT! My favorite is `useStuckness` 😂. I don't if you did the same since `useIsStickyStuck` appears in my list as well. I personally like  `usePositionStickyObserver` (not in this list but inspired by the list) which fits nicely with our other "useSomethingObserver" compositions. 

useStickyState
useIsElementStuck
useStickyPositionCheck
useStuckStatus
useStickyElementWatcher
useStickyRefState
useIsStickyStuck
useCheckStickyStatus
useStickyPositionObserver
useStickyElementState
useStickySituation
useStuckElementDetector
useStickyStatusMonitor
useStickyPositionTracker
useStuckness
useIsRefStuck
useStickyRefWatcher
useStuckStateObserver
useStickyElementChecker
useStuckPositionMonitor

Reply:
Ah `usePositionStickyObserver` for the win for sure, especially when returning an object that we might provide more observation values on in the future.